buildlib/tests.yml

```diff
@@ -35,6 +25,8 @@ jobs:
           worker: $(worker_id)
           BUILD_NUMBER: "$(Build.BuildId)-$(Build.BuildNumber)"
           JOB_URL: "$(System.TeamFoundationCollectionUri)$(System.TeamProject)/_build/results?buildId=$(Build.BuildId)"
-          EXECUTOR_NUMBER: $(executor_number)
-          JENKINS_RUN_TESTS: 1
+          JENKINS_RUN_TESTS: ""
```

Comment:
can we remove JENKINS_RUN_TESTS altogether?

Reply:
Currently setting affinity depends on this parameter: https://github.com/openucx/ucx/blob/5ef72d9f2a3648b19b80333c1ff496987173b9d1/contrib/test_jenkins.sh#L49-L54

Will have to rework it too.

Reply:
@yosefe what logic should be for applying affinity? Right now it is set when both `$EXECUTOR_NUMBER` and `$JENKINS_RUN_TESTS` are set.

Reply:
yes, today if JENKINS_RUN_TESTS is not set we don't apply affinity (assuming it runs in a VM)
otherwise it's according to EXECUTOR_NUMBER

Reply:
i guess we can keep it this way and let it inherit affinity when JENKINS_RUN_TESTS is not set

Reply:
So we can go with this PR or you want me to change it?

Reply:
can have this PR. can you pls add code comment in here to explain why we do this?

Reply:
comment: set JENKINS_RUN_TESTS to empty value to avoid setting CPU affinity in test_jenkins.sh